chore: update block proposal errors dashboard panel

nflaig opened the pull request:
Update `operationId` in block proposal errors dashboard panel to use latest version of apis as previous (deprecated) apis are unused in Lodestar by default (and soon to be removed).

nflaig committed:
chore: update block proposal errors dashboard panel

nflaig commented on packages/api/src/beacon/routes/beacon/block.ts:
```diff
@@ -499,7 +499,6 @@ export function getDefinitions(config: ChainForkConfig): RouteDefinitions<Endpoi
           const fork = config.getForkName(signedBlindedBlock.message.slot);
           return {
             body: getExecutionForkTypes(fork).SignedBlindedBeaconBlock.toJson(signedBlindedBlock),
-
```
unrelated but might as well fix formatting here

twoeths commented:
@nflaig would be great to fix broken "Lodestar - block production" panel "Success + Error" rates, labels are duplicated

<img width="1334" alt="Screenshot 2024-10-21 at 09 09 19" src="https://github.com/user-attachments/assets/090f7792-e7cf-481a-a04a-5ed515945f74">

nflaig commented:
There are also "dead" panels as we have removed metrics for some of the panels, e.g. all the 3 empty panels here

![image](https://github.com/user-attachments/assets/9e086af4-44cc-4224-82ea-c5999c42708c)

We have added a different metrics for submitted block proposals in validator monitor which we can use to replace those, also wanna take a look at the local vs. builder %, I thought we had a panel but does not seem to be the case, also noted here https://github.com/ChainSafe/lodestar/issues/6108 that it did not work as expected

wemeetagain commented:
:tada: This PR is included in [v1.23.0](https://github.com/ChainSafe/lodestar/releases/tag/v1.23.0) :tada:
